devops/tests/test_component.py

```diff
@@ -78,3 +103,23 @@ def test_patch_replicas():
     c._patch_replicas(deploy)
 
     assert deploy["spec"]["replicas"] == 77
+
+
+def test_patch_cronjob():
+    cronjob = get_cronjob()
+    c = Component("service/test-cronjob")
+    c.replicas = 77
```

Comment:
I started wondering that what replicas would actually mean for a cron job. I couldn't find any reference to it at least on https://kubernetes.io/docs/tasks/job/automated-tasks-with-cron-jobs/#writing-a-cron-job-spec. Could you @victorv-sbp please point me to some documentation about that?

Reply:
Indeed, the replicas is nowhere to be found on the kubernetes documentation regarding Jobs.
We've supposed that replicas would just create more pods on job creation/trigger (as in deployment case) as the kubeval reported the spec with replicas as a valid one.

However, after more in dept research we've come to the conclusion that the replicas is not needed in a job definition.

We've changed the code accordingly, please review the changes when you get a chance.

Reply:
Looks good to me.